### ISSUE TITLE
ISR: Make ISR work with the new SSR setup in Angular

### ISSUE THREAD

**Issue by eneajaho** [rx-angular/rx-angular#1800]

**Comment by maxisam** [comment]
Do they change anything in Angular 19?

**Comment by maxisam** [comment]
https://github.com/JeanMeche/ssr-v19

I guess this is what you are talking about?

**Comment by eneajaho** [comment]
@maxisam Yeah, but it has always been in my mind to make the ISR more independent from what the Angular provides and make it easier to migrate to latest versions.

**Comment by maxisam** [comment]
@eneajaho all my features passed our internal testing and some playwright e2e testing. 

Here is our company site: https://www.mpix.com/ 

You can see the comment from the source saying we are using ISR 😆

**Comment by vbulat** [comment]
Hi. Any updates on this topic ?

**Comment by eneajaho** [comment]
I have a POC idea here https://github.com/eneajaho/new-isr-poc/blob/main/src/server.ts 
But I haven't had any time to put into this one currently.

**Comment by LeParadoxHD** [comment]
Any update?

**Comment by SkyZeroZx** [comment]
Thank you for all your work at ISR Angular.
This could be interesting (I'll personally see if I can review/investigate to add compatibility). Since `CommonEngine` will be deprecated in the next version of Angular: 
See PR https://github.com/angular/angular-cli/pull/33127
https://github.com/angular/angular-cli/commit/50b16a65b1be1f9c2ec11d578240a8884518d517

EDIT:  It seems it was already compatible, apparently it's just not documented
